<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMdeportee</class>
 <widget class="QMainWindow" name="IHMdeportee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHMdeportee</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="patient">
     <attribute name="title">
      <string>Patient</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxPatient">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>581</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Informations patients </string>
      </property>
      <widget class="QLabel" name="labelNom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPrenom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Prénom</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelDate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de la manipulation</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Numéro de la manipulation</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateManip">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>140</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>90</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="numManip">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="suivantPatient">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>300</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="heure">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>430</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reglages1">
     <attribute name="title">
      <string>Réglages</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxReglages1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>581</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Paramètres impulsions</string>
      </property>
      <widget class="QSlider" name="largeurImpulsion">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>90</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
      <widget class="QSlider" name="largeurFenetre">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="periodicite">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="amplitudeMax">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="nombreImpulsions">
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelNombreImpulsion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre d'impulsions dans la fenêtre</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLargeurImpulsion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Largeur des impulsions </string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLargeurFenetre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Largeur fenêtre </string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPeriode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Périodicité du train d'impulsions</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelAmplitudeMax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Amplitude maximale de I2 (mA)</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeurLargeurImpulsion">
       <property name="geometry">
        <rect>
         <x>411</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="valeurLargeurFenetre">
       <property name="geometry">
        <rect>
         <x>411</x>
         <y>130</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="valeurPeriode">
       <property name="geometry">
        <rect>
         <x>411</x>
         <y>180</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="valeurAmplitude">
       <property name="geometry">
        <rect>
         <x>411</x>
         <y>230</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="suivantReglages1">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="precedentReglages1">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Précédent</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="reglages2">
     <attribute name="title">
      <string>Réglages</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxReglages2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>581</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Paramètres impulsions</string>
      </property>
      <widget class="QPushButton" name="suivantReglages2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="precedentReglages2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Précédent</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="resume">
     <attribute name="title">
      <string>Résumé</string>
     </attribute>
     <widget class="QLabel" name="labelResume">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>340</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Commencer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="precedentResume">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Précédent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="simulation">
     <attribute name="title">
      <string>Graphique</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="reprendre">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reprendre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Arrêt 
d'urgence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="batterie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="chronometre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>largeurImpulsion</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>valeurLargeurImpulsion</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largeurFenetre</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>valeurLargeurFenetre</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodicite</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>valeurPeriode</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amplitudeMax</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>valeurAmplitude</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
